diff --git a/node_modules/react-native-progress-circle/src/index.js b/node_modules/react-native-progress-circle/src/index.js
index 50cb271..0c4a0e5 100644
--- a/node_modules/react-native-progress-circle/src/index.js
+++ b/node_modules/react-native-progress-circle/src/index.js
@@ -1,11 +1,11 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { StyleSheet, View, ViewPropTypes,I18nManager } from 'react-native'
+import { StyleSheet, View, ViewStyle, ViewPropTypes,I18nManager } from 'react-native'
 
 // compatability for react-native versions < 0.44
 const ViewPropTypesStyle = ViewPropTypes
   ? ViewPropTypes.style
-  : View.propTypes.style
+  : ViewStyle
 let direction = I18nManager.isRTL? 'right' : 'left';
 const styles = StyleSheet.create({
   outerCircle: {
